studio/components/interfaces/TableGridEditor/SidePanelEditor/RowEditor/InputField.tsx

```diff
@@ -72,7 +72,7 @@ const InputField: FC<Props> = ({
         error={errors[field.name]}
         onChange={(event: any) => onUpdateField({ [field.name]: event.target.value })}
         actions={
-          <Button type="default" onClick={onViewForeignKey} icon={<IconLink />}>
+          <Button type="default" htmlType="button" onClick={onViewForeignKey} icon={<IconLink />}>
```

Comment:
that's odd, htmlType is already button by default

Reply:
Doesn't look like it: https://github.com/supabase/ui/blob/master/src/components/Button/Button.tsx#L63

Reply:
not yet i think?
https://github.com/supabase/ui/blob/81da906446068eec64910eb29d1cefdc7a3ee30f/src/components/Button/Button.tsx#L67